### PR TITLE
add data science exercise

### PR DESCRIPTION
**Pull request by miguelsimon**
This is meant as a simple exercise to hone data science related skills, namely:

* transforming data using basic sql
* using basic visualizations to understand the data
* understanding what an observational study is
* the dangers of jumping to causal conclusions when interpreting observational studies

### CODE REVIEW

File: data_science_1/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+* `hours_per_day`: horas de ejercicio por día medias que realiza el participante
+* `mmol_per_l`: el nivel de thanatol medido en sangre del paciente, en milimoles por litro
+
+#### ejercicio 1
+
+Queremos combinar los datos de los pacientes adultos y menores de edad para poder tratarlos de forma homogénea.
+
+Crea una view que contenga las siguientes columnas para todos los participantes, tanto menores como adultos:
+
+* age: edad del participante en años en el momento del estudio
```
Comment (valentinboyanov) [inline data_science_1/README.md]: Minor formatting consistency issue about column names, eg: `age` will be consistent.

**Review by valentinboyanov — CHANGES_REQUESTED** [review]
I've been able to understand the exercise, links work and columns and data seem to be OK:

```
data_science_1 git:(data_science_1) sqlite3
SQLite version 3.39.5 2022-10-14 20:58:05
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
sqlite> .read dataset_3.sql
sqlite> .tables
ages                 exercise             pediatric          
data_collection_day  participants         thanatol           
sqlite> .headers on
sqlite> select * from ages limit 1;
id|birthday
767|2005-04-04
sqlite> select count(*) from ages;
count(*)
963
sqlite> select * from exercise limit 1;
id|hours_per_week
767|5.89889879900017
sqlite> select count(*) from exercise;
count(*)
963
sqlite> select * from participants limit 1;
id
0
sqlite> select count(*) from participants;
count(*)
963
sqlite> select * from data_collection_day limit 1;
data_collection_day
2023-03-31
sqlite> select count(*) from data_collection_day;
count(*)
1
sqlite> select * from thanatol limit 1;
id|mmol_per_l
767|14.3373875132598
sqlite> select count(*) from thanatol;
count(*)
963
sqlite> select * from pediatric limit 1;
age|hours_per_day|mmol_per_l
16.0|0.834732072023712|14.1136291834294
sqlite> select count(*) from pediatric;
count(*)
37
sqlite> 
```

I've found only one minor formatting issue, otherwise lgtm!

**Comment by valentinboyanov** [comment]
Not so related to this PR, but I've noticed that the main index is somewhat inconsistent:

https://github.com/miguelsimon/ejercicios/blob/7eae227f11ffe59a6b62285fd3d7a2297e9738fa/README.md?plain=1#L7-L21

**Comment by miguelsimon** [comment]
I've noticed the inconsistency in the main index too but I'm leaving it alone for now as I don't really have a good idea for a unifying style yet.

**Review by valentinboyanov — APPROVED** [review]
lgtm!

**Pull request merged**